src/main/webapp/WEB-INF/view/searchResults.jsp

```diff
@@ -38,10 +39,11 @@
 	<ol class="list-unstyled col-md-8">
 		<c:forEach var="doc" varStatus="s" items="${documentItems}">
 			<li id="result${s.index}">
-				<h3 class="title text-truncate">
+				<h3 class="title text-truncate" style="margin-bottom:0;">
 					<a class="link" href="${doc.url_link}" data-uri="${doc.url_link}"
 						data-id="${doc.doc_id}" data-order="${s.index}">${doc.content_title}</a>
-				</h3>
+				</h3>									
+				<p>${doc.url}</p>
```

Comment:
please remove this p tag.

Reply:
It was an unnecessary code. We have deleted it.
My apologies for the residual debugging code.